translations/translations.json

```diff
@@ -3871,7 +3877,10 @@
       "shouldBeEarlier": "The start date should be before the end date.",
       "hideFilters": "Hide filters",
       "showFilters": "Show filters",
-      "selectRange": "Select Range"
+      "selectRange": "Select Range",
+      "dd": "dd",
+      "mm": "mm",
+      "yyyy": "yyyy"
```

Comment:
🔧 To be honest, I'm not even sure this is correct in English. I haven't really found any websites or apps that let you type in the values though, so I'm not sure. But I would assume that most users want a "day", "month", "year" there.